haystack/modeling/data_handler/processor.py

```diff
@@ -2241,8 +2239,26 @@ def _read_squad_file(filename: str, proxies=None):
     return input_data
 
 
-def http_get(url, temp_file, proxies=None):
-    req = requests.get(url, stream=True, proxies=proxies)
+def http_get(
+    url: str,
+    temp_file: IO[bytes],
+    proxies: Optional[Dict[str, str]] = None,
+    timeout: Union[float, Tuple[float, float]] = 10.0,
+):
+    """
+    Runs a HTTP GET requests and saves response content to file.
+    :param url: URL address
+    :type url: str
```

Comment:
No need to add the types here as we already have them in the type annotations in the methods signature